catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.config.xml

```diff
@@ -63,22 +63,6 @@
             default=""
             required="false"/>
 
-        <AD id="spacingMode"
```

Comment:
❗️ Does the documentation need to be updated for all of the configuration changes?

Reply:
Updated Documentation

Reply:
![image](https://user-images.githubusercontent.com/7236299/43154475-4e04a448-8f29-11e8-8702-ad9d69949f36.png)
![image](https://user-images.githubusercontent.com/7236299/43154489-55adf302-8f29-11e8-80ba-c330eed9c3eb.png)